ledfx/effects/audio.py

```diff
@@ -334,10 +344,6 @@ def open_audio_stream(device_idx):
                     callback=self._audio_sample_callback,
                     dtype=np.float32,
                     latency="low",
-                    blocksize=int(
-                        device["default_samplerate"]
-                        / self._config["sample_rate"]
-                    ),
```

Comment:
https://python-sounddevice.readthedocs.io/en/0.5.0/api/streams.html#sounddevice.Stream

blocksize (int, optional) – The number of frames passed to the stream callback function, or the preferred block granularity for a blocking read/write stream. The special value blocksize=0 (which is the default) may be used to request that the stream callback will receive an optimal (and possibly varying) number of frames based on host requirements and the requested latency settings.

We will need to do a bit of testing on this, but nothing has exploded for me.